src/pages/Folder/components/FolderCategory/index.jsx

```diff
@@ -1,49 +1,52 @@
+import { useModal } from "hooks/useModal";
 import styles from "./styles.module.css";
-import useWindowSize from "hooks/useWindowSize";
+import { AddModal, ModalPortal } from "components";
 
-export default function FolderCategory({
-  folders,
-  selectedId,
-  onSelectedFolder,
-}) {
-  const [windowSize] = useWindowSize();
+export function FolderCategory({ folders, selectedId, onSelectedFolder }) {
+  const { openModal, modalRef, handleModalClose, handleModalOpen } = useModal();
 
-  const handleClick = (e) => {
+  const handleFolderSelectClick = (e) => {
     onSelectedFolder({ name: e.target.textContent, id: e.target.id });
   };
 
+  const handleFolderAddClick = (e) => {
+    handleModalOpen();
+  };
```

Comment:
P3:
불필요한 매개변수 지워주세요.

```suggestion
  const handleFolderAddClick = () => {
    handleModalOpen();
  };
```